Resources/Locale/en-US/chat/managers/chat-language.ftl

```diff
@@ -0,0 +1,15 @@
+chat-language-Universal-name = Universal
```

Comment:
That will be hard to maintain. A person adding a new language will not know there's another file they have to modify. Ideally, all of this should be placed in the respective language locale files.